api/interface.py

```diff
@@ -410,6 +410,22 @@ class CommandKeyboardConfig(BaseModel):
     release: Optional[bool] = None
     """Whether to release the key. Optional."""
 
+class AudioFile(BaseModel):
+    path: str
+    """The audio file to play. Required."""
+    
+    name: str[Optional]
+    """The name of the audio file. Optional."""
+
+class AudioFileConfig(BaseModel):
+    file: AudioFile
+    """The audio file to play. Required."""
+
+    volume: Optional[float] = None
```

Comment:
Let's stick to our usual pattern: A property is only optional if it isn't required at all. Else we'll have to always check for `if volume` and differentiate between `volume == 0` and `not volume`. Make this non-optional and default it to 1 when you create an instance of it.

Reply:
Made non-Optional